src/transformStateWithClones.js

```diff
@@ -7,7 +7,40 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArray = [];
+  let currentState = { ...state }; // Creating a copy of the initial state
```

Comment:
Why do you write in the comment that this is a copy of the state, but call it the `currentState`? 
Then it would be better to call this variable `copiedState`.
Also, this comment doesn't look very useful, because here the code is "self-explanatory".